src/js/components/CreatureAbility.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+export default class CreatureAbility extends React.Component {
+
+  constructor(props) {
+    super(props);
+    
+    this.handleValueChanged = this.handleValueChanged.bind(this);
+  }
+
+  handleValueChanged(key, event) {
+    this.props.onChange(key, event.target.value);
+  }
+
+  renderDisplay() {
+    return (
+      <div className='creatureAbility'>
+        <span className='abilityName'>{this.props.name}.</span>
+        <span className='abilityDescription'>{this.props.description}</span>
```

Comment:
always a good practice to deconstruct the props and state you are going to use.
`const { name, description } = this.props`
not you can use them like regular const here

Reply:
Functionally there isn't much difference but for readability and maintainability remove extra overhead of dereferencing props goes a long way.
Imaging `this.props` or `this.state` showing up repeatedly in your function.
Also `const {name, description} = this.props;` documents that you will be using only these two properties from the props. Rest of the function doesn't need to be aware of the fact that these came from `props`.